Mathlib/Probability/Kernel/Condexp.lean

```diff
@@ -213,4 +214,60 @@ theorem condexp_ae_eq_integral_condexpKernel [NormedAddCommGroup F] {f : Ω →
     μ[f|m] =ᵐ[μ] fun ω => ∫ y, f y ∂condexpKernel μ m ω :=
   ((condexp_ae_eq_integral_condexpKernel' hf_int).symm.trans (by rw [inf_of_le_left hm])).symm
 
+section Cond
+
+/-! ### Relation between conditional expectation, conditional kernel and the conditional measure. -/
+
+variable {s t : Set Ω} [NormedAddCommGroup F] [NormedSpace ℝ F] [CompleteSpace F]
+
+omit [StandardBorelSpace Ω]
+
+lemma condexp_generateFrom_singleton (hs : MeasurableSet s) {f : Ω → F} (hf : Integrable f μ) :
```

Comment:
`hf : Integrable f μ` is needed because if `f` is not integrable the lhs is 0 but the rhs might be nonzero because we could have `hf : IntegrableOn f s μ`, right?